test/mini_racer_test.rb

```diff
@@ -872,6 +872,14 @@ def test_pipe_leak
     end
   end
 
+  def test_capture_ruby_exception_message
+    context = MiniRacer::Context.new()
+    error_message = "Actual Error Message"
+    context.attach("a", proc{|a| raise error_message})
+
+    assert_equal error_message, context.eval("try { a(); } catch (e) { e }")
```

Comment:
Isn't it rather weird that the return value of this is a JS exception, but the return value of this `context.eval` is just a string and not an exception? I think that's pretty confusing.
It seems like it should at least be wrapped into some MiniRacer exception or so.